sdk/search/azure-search-documents/swagger/README.md

```diff
@@ -67,51 +70,10 @@ These settings apply only when `--tag=package-2020-06-searchindex` is specified
 ``` yaml $(tag) == 'package-2020-06-searchindex'
 namespace: com.azure.search.documents
 input-file:
-- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/0bc7853cb4d824bb6c310344dcc1b5f77cbe6bdd/specification/search/data-plane/Azure.Search/preview/2020-06-30/searchindex.json
+- https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/search/data-plane/Azure.Search/preview/2020-06-30/searchindex.json
```

Comment:
the only change is "0bc7853cb4d824bb6c310344dcc1b5f77cbe6bdd" to "master"?

Reply:
This was changed as it should be targeting master instead of a specific commit. The other Swagger configuration is targeting a specific commit as the changes haven't been merged yet.